src/content/posts/thanks.mdx

```diff
@@ -4,6 +4,7 @@ author: Andreas Kling
 description: Place holder.
 image: /assets/img/plant.webp
 type: Hidden
+noindex: true
```

Comment:
This isn't getting propagated through to the header file. But also, this may not be needed, as you could just hide the index based on the `type` and do noindex if it's `type: Hidden`. Either way, it somehow needs to get passed to the header on the posts page.

Reply:
Fair enough. 

I did consider reusing `Hidden` for that, but I thought there might be pages you want hidden from menus, but still indexed in search. 

I also noticed you can access `https://ladybird.org/newsletter/2024-07-31/` but they are not indexed - do you want them to be?